x/crosschain/types/expected_keepers.go

```diff
@@ -11,6 +11,8 @@ import (
 type StakingKeeper interface {
 	GetAllValidators(ctx sdk.Context) (validators []stakingtypes.Validator)
 	GetValidator(ctx sdk.Context, addr sdk.ValAddress) (validator stakingtypes.Validator, found bool)
+	SetParams(ctx sdk.Context, params stakingtypes.Params)
+	GetParams(ctx sdk.Context) (params stakingtypes.Params)
```

Comment:
I would add a TODO to remember to remove those once we revert changes

Reply:
I dont think we are planning to merge this PR , we can go to v7.0.0 directly